recipe/meta.yaml

```
@@ -1,80 +1,218 @@
+        - if not exist "%LIBRARY_BIN%\cudnn_adv64_{{ soname }}.dll" (echo Missing cudnn_adv64_{{ soname }}.dll && exit /b 1)  # [win]
+        - if not exist "%LIBRARY_BIN%\cudnn_cnn64_{{ soname }}.dll" (echo Missing cudnn_cnn64_{{ soname }}.dll && exit /b 1)  # [win]
+        - if not exist "%LIBRARY_BIN%\cudnn_engines_precompiled64_{{ soname }}.dll" (echo Missing cudnn_engines_precompiled64_{{ soname }}.dll && exit /b 1)  # [win]
+        - if not exist "%LIBRARY_BIN%\cudnn_engines_runtime_compiled64_{{ soname }}.dll" (echo Missing cudnn_engines_runtime_compiled64_{{ soname }}.dll && exit /b 1)  # [win]
+        - if not exist "%LIBRARY_BIN%\cudnn_graph64_{{ soname }}.dll" (echo Missing cudnn_graph64_{{ soname }}.dll && exit /b 1)  # [win]
+        - if not exist "%LIBRARY_BIN%\cudnn_heuristic64_{{ soname }}.dll" (echo Missing cudnn_heuristic64_{{ soname }}.dll && exit /b 1)  # [win]
+        - if not exist "%LIBRARY_BIN%\cudnn_ops64_{{ soname }}.dll" (echo Missing cudnn_ops64_{{ soname }}.dll && exit /b 1)  # [win]
+        - test -f $PREFIX/lib/libcudnn.so.{{ majorminorpatch }}                                      # [linux]
+        - test -f $PREFIX/lib/libcudnn.so.{{ soname }}                                               # [linux]
+        - test -f $PREFIX/lib/libcudnn_adv.so.{{ majorminorpatch }}                                  # [linux]
+        - test -f $PREFIX/lib/libcudnn_adv.so.{{ soname }}                                           # [linux]
+        - test -f $PREFIX/lib/libcudnn_cnn.so.{{ majorminorpatch }}                                  # [linux]
+        - test -f $PREFIX/lib/libcudnn_cnn.so.{{ soname }}                                           # [linux]
+        - test -f $PREFIX/lib/libcudnn_engines_precompiled.so.{{ majorminorpatch }}                  # [linux]
+        - test -f $PREFIX/lib/libcudnn_engines_precompiled.so.{{ soname }}                           # [linux]
+        - test -f $PREFIX/lib/libcudnn_engines_runtime_compiled.so.{{ majorminorpatch }}             # [linux]
+        - test -f $PREFIX/lib/libcudnn_engines_runtime_compiled.so.{{ soname }}                      # [linux]
+        - test -f $PREFIX/lib/libcudnn_graph.so.{{ majorminorpatch }}                                # [linux]
+        - test -f $PREFIX/lib/libcudnn_graph.so.{{ soname }}                                         # [linux]
+        - test -f $PREFIX/lib/libcudnn_heuristic.so.{{ majorminorpatch }}                            # [linux]
+        - test -f $PREFIX/lib/libcudnn_heuristic.so.{{ soname }}                                     # [linux]
+        - test -f $PREFIX/lib/libcudnn_ops.so.{{ majorminorpatch }}                                  # [linux]
+        - test -f $PREFIX/lib/libcudnn_ops.so.{{ soname }}                                           # [linux]
+        - ${GCC} test_load_elf.c -std=c99 -Werror -ldl -o test_load_elf                              # [linux]
```

Who defines `GCC`?  And clearly it works on `osx` -- hopefully not picking up `/usr/bin/gcc`...

```suggestion
        - ${CC} test_load_elf.c -std=c99 -Werror -ldl -o test_load_elf                              # [linux]
```